cli/args.go

```diff
@@ -82,7 +82,7 @@ func parseTerragruntOptionsFromArgs(args []string, writer, errWriter io.Writer)
 
 	opts.TerraformPath = filepath.ToSlash(terraformPath)
 	opts.AutoInit = !parseBooleanArg(args, OPT_TERRAGRUNT_NO_AUTO_INIT, os.Getenv("TERRAGRUNT_AUTO_INIT") == "false")
-	opts.NonInteractive = parseBooleanArg(args, OPT_NON_INTERACTIVE, false)
+	opts.NonInteractive = parseBooleanArg(args, OPT_NON_INTERACTIVE, os.Getenv("TF_INPUT") == "true")
```

Comment:
Shouldn't this be `os.Getenv("TF_INPUT") == "false"`? Also, [according to the docs](https://www.terraform.io/docs/configuration/environment-variables.html#tf_input), `0` is also valid.